### PR TITLE
Include milestone link at the top of github release notes

### LINKED ISSUE #60: Include milestone link in github release notes

### PR DESCRIPTION
**Pull request by mathias-baumann-sociomantic**
fixes #60

### CODE REVIEW

File: src/release/main.d [hunk 1]
```diff
@@ -173,6 +172,40 @@ void main ( string[] params )
...
+    import octod.api.releases;
+    import std.algorithm;
+    import std.stdio;
+    import std.range;
+
+    auto mstone_list = listMilestones(con, repo).find!(a=>a.title == ver);
+
+    string mstone_link;
+
+    if (!mstone_list.empty)
```
Comment (nemanja-boric-sociomantic) [inline src/release/main.d]: Do you want to check if there's only one, or just use the first one?

Comment (leandro-lucarella-sociomantic) [inline src/release/main.d]: Maybe report a warning if you find more than one (which could happen), or ideally let the user pick one (but might be too much effort for what it should be an error in neptune's ways).

**Comment by nemanja-boric-sociomantic** [comment]
I agree with @leandro-lucarella-sociomantic

File: src/release/main.d (hunk 1 above)
Comment (mihails-strasuns-sociomantic) [inline src/release/main.d]: Can it really happen? You can't create multiple milestones with the same name and find predicate looks for exact match. I'd simply enforce it.

**Comment by mathias-baumann-sociomantic** [comment]
I went the lazy route now and added an enforce as suggested by mihails

**Pull request merged**